src/containerapp/HISTORY.rst

```diff
@@ -5,6 +5,10 @@ Release History
 upcoming
 ++++++
 
+1.2.0b2
```

Comment:
Please don't change the version here.
Generally we have a separate PR to update the version and release